### PR TITLE
Fix docstrings, typing and string construction - controller

### LINKED ISSUE #88: Fix docstrings, typing and string construction - controller

### PR DESCRIPTION
**Pull request by IKostric**
Fixes #88

### CODE REVIEW

File: moviebot/controller/controller_telegram.py [hunk 1]
```diff
@@ -33,11 +34,9 @@
...
 
 class ControllerTelegram(Controller):
-    """This is the Controller class which controls the flow of the conversation
-    while the user interacts with the agent using telegram."""
-
-    def __init__(self):
-        """Initializes some basic structs for the Controller."""
+    def __init__(self) -> None:
+        """Controller class which controls the flow of the conversation while
+        the user interacts with the agent using telegram."""
```
Comment (WerLaj) [inline moviebot/controller/controller_telegram.py:39]: Can we try to fit it in one line? For example, _Controller class to control the flow of the conversation on Telegram._

**Review by WerLaj — APPROVED** [review]
LGTM with some minor comments

File: moviebot/controller/controller_telegram.py (hunk 1 above)
Comment (IKostric) [reply moviebot/controller/controller_telegram.py:39]: done

File: moviebot/controller/controller_telegram.py [hunk 2]
```diff
@@ -71,14 +75,19 @@ def load_bot_token(self, bot_token_path):
...
+        system before.
 
         Args:
-            update:
-            context:
+            update: The update object from Telegram.
+            context: The context object from Telegram.
+
+        Returns:
+            CONTINUE: The continue state of the conversation.
```
[the remark this answers is not in the document]
Comment (IKostric) [reply moviebot/controller/controller_telegram.py:90]: done

File: moviebot/controller/controller_telegram.py [hunk 3]
```diff
@@ -122,26 +131,32 @@ def start(self, update, context):
...
         """Sends the users the instructions if they ask for help.
 
         Args:
-            update:
-            context:
+            update: The update object from Telegram.
+            context: The context object from Telegram.
+
+        Returns:
+            CONTINUE: The continue state of the conversation.
```
[the remark this answers is not in the document]
Comment (IKostric) [reply moviebot/controller/controller_telegram.py:142]: done

File: moviebot/controller/controller_telegram.py [hunk 4]
```diff
@@ -122,26 +131,32 @@ def start(self, update, context):
...
         the uses to begin with.
 
         Args:
-            update:
-            context:
+            update: The update object from Telegram.
+            context: The context object from Telegram.
+
+        Returns:
+            CONTINUE: The continue state of the conversation.
```
[the remark this answers is not in the document]
Comment (IKostric) [reply moviebot/controller/controller_telegram.py:159]: done

File: moviebot/controller/controller_telegram.py [hunk 5]
```diff
@@ -183,12 +198,16 @@ def restart(self, update, context):
...
         """Continues the conversation until the users want to restart of exit.
 
         Args:
-            update:
-            context:
+            update: The update object from Telegram.
+            context: The context object from Telegram.
+
+        Returns:
+            The continue status of the conversation or the end of conversation
```
[the remark this answers is not in the document]
Comment (IKostric) [reply moviebot/controller/controller_telegram.py:209]: done

File: moviebot/controller/controller_telegram.py [hunk 6]
```diff
@@ -270,12 +289,15 @@ def continue_conv(self, update, context):
...
         """Exit the conversation.
 
         Args:
-            update:
-            context:
+            update: Object representing an incoming update.
+            context: Context object.
+
+        Returns:
+            ConversationHandler.END
```
[the remark this answers is not in the document]
Comment (IKostric) [reply moviebot/controller/controller_telegram.py:300]: done

**Pull request merged**